### DISCUSSION TITLE
ABI dashboard up and ready.

### DISCUSSION THREAD

**Discussion by lgirdwood** [thesofproject/sof#7376] [Team Posts]
All,  ABI dashboard is now up and running. New ABI updates should automatically go to TSC for review.  Fwiw, we have had some MINOR update races between FW and driver as can be seen
in the ABI dashboard :-

https://github.com/orgs/thesofproject/projects/2

ABI MINOR 6 is the synchronization point as FW and driver align at supported
features at this level.

Going forward all PRs will be added to the dashboard and classified.

MINOR 7 will be test IPC class.

MINOR 8+ unclassified as all PRs at RFC level.

**Comment by jajanusz** [comment]
I need some clarification on the process here.
What if I have PR from newer version ready before older one?
I assume, I cannot merge it because there would make no sense to bump something f.e. to api 8 when there are still pending changes from 7.
Let's assume I have something like this:
```
ABI 1 - PR A, B
ABI 2 - PR C, D
```
`PR A` get merged and abi is bumpbed to 1, then I have `PR C` ready to be merged, but I shouldn't do that cos `PR B` is still WIP. Is it ok to move `PR B` to `ABI 2`, so I can keep merging other ABI-changing PRs?

**Comment by lgirdwood** [comment]
@jajanusz we can group PRs together as a single MINOR. If a PR is taking too long then we can increase it's MINOR and merge it later. Yes, feel free to manage the PR MINORs to take into consideration of developer progress and PR readyness for merge.
